src/dashboard/get_chart_data/get_chart_data.py

```diff
@@ -196,6 +198,7 @@ def _build_query(query_params: dict, filter_groups: list, path_params: dict) ->
             coalesce_columns=columns,
             inline_configs=inline_configs,
             none_configs=none_configs,
+            extra_filter_configs=[],  # extra_filter_configs
```

Comment:
What's this line about? Did you mean to leave that comment like that?

Reply:
That was from an in process attempt to fix the underlying ginger template, which moved away from; will delete